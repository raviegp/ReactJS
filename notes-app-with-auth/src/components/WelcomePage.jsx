import React, { Fragment } from 'react';
import ImgUrl from '../images/notes-background.png';

const WelcomePage = () => (
    <Fragment>
        <div align='center'>
            <h2 style={{ color: 'green' }}>
                Welcome to Notes App!!!. Please login to see the features!
            </h2>          
        </div>
        <img src={ImgUrl} alt="Notes App"  height="100%" width="100%"/>              
    </Fragment>
);

export default WelcomePage;