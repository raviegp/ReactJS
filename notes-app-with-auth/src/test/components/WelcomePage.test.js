import React from 'react';
import { shallow } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';
import WelcomePage from '../../components/WelcomePage';
 
expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));

test('test welcome page', () => {
    const wrapper = shallow(<WelcomePage />);
    expect(wrapper).toMatchSnapshot();
});