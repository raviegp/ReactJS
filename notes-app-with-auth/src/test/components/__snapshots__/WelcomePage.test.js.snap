// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test welcome page 1`] = `
<Fragment>
  <div
    align="center"
  >
    <h2
      style={
        Object {
          "color": "green",
        }
      }
    >
      Welcome to Notes App!!!. Please login to see the features!
    </h2>
  </div>
  <img
    alt="Notes App"
    height="100%"
    src="notes-background.png"
    width="100%"
  />
</Fragment>
`;
